app.go

```diff
@@ -397,3 +433,18 @@ func (a *App) CheckLSPClosedChannelMismatch(
 	}
 	return &data.CheckLSPClosedChannelMismatchResponse{Mismatch: mismatch}, nil
 }
+
+func (a *App) EnableOrDisableTor(enable bool) error {
```

Comment:
The names EnableOrDisableTor and IsTorActive are not consistent. I suggest to use in all places SetTor() and GetTor() where the SetTor parameter and the return value of GetTor is an enum constant from the breez/tor package (use the first iota as undefined because it's the default value).

Reply:
If it's okay with you, can I do this *after* all code review is complete and someone other than myself has tested tor satisfactorily and @kingonly has verified that it's feature complete?

(Right now, whenever I change `api.go` my build becomes fragile and breaks. I'm not yet sure if I need to reinstall Android Studio or roll back an upgrade or if something else is the matter. But it makes me not want to change the interface for now while everything is still working correctly.)